app/Jobs/ElasticSearchIndexDelete.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace App\Jobs;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use App\WikiSetting;
+use App\Http\Curl\CurlRequest;
+use App\Http\Curl\HttpRequest;
+use App\Wiki;
+
+class ElasticSearchIndexDelete extends Job implements ShouldBeUnique
+{
+    private $wikiId;
+
+    private $request;
+
+    /**
+     * @return void
+     */
+    public function __construct( int $wikiId, HttpRequest $request = null )
```

Comment:
Will have the same HttpRequest $request injection issue

Reply:
done